sample-app/build.gradle

```diff
@@ -65,8 +46,12 @@ android {
 }
 
 dependencies {
+//    this line tests against the local forage-android
     implementation(project(":forage-android"))
 
+//     uncomment this line 👇 to test against remote repository forage-android
+//    implementation("com.joinforage:forage-android:3.1.0.1")
```

Comment:
is it supposed to be `3.1.0` ?

Reply:
This was my attempt to make a version that GoPuff would not try to install while I was testing whether my changes to `CI.yaml` would work. I left it as a "placeholder" so to speak, so folks would replace it with any version they want to install from the repository. Clearly, the comment + fake version is not expressing my intent, so let me update the code to try and make the intent more clear. Thanks for flagging!

Reply:
Addressed in [f2b6450](https://github.com/teamforage/forage-android-sdk/pull/114/commits/f2b6450d4915b1dcb1c22b2d5f5257bb544cd433)